bucket/hydrus-network.json

```
@@ -0,0 +1,51 @@
+{
+    "version": "373",
+    "description": "A personal booru-style media tagger",
+    "homepage": "https://hydrusnetwork.github.io/hydrus/",
+    "license": "WTFPL",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/hydrusnetwork/hydrus/releases/download/v373/Hydrus.Network.373.-.Windows.-.Extract.only.zip",
+            "hash": "5dd7b27159e366a846f293ccadaf48f6a8b5b05ad963e49cbacf8f373751b40a"
+        }
+    },
+    "extract_dir": "Hydrus Network",
+    "post_install": "Get-ChildItem \"$persist_dir\\db\" -File | Copy-Item -Destination \"$dir\\db\" -Force",
```

Why post_install? Just for skipping existence check?

---

Yes, it is. Maybe it's better to persist the whole db directory and copy sqlite3.exe from db.origin?

---

The sqlite3.exe inside the db directory is not essential to the program, it comes with hydrus network as a way for users to recover the db if hydrus itself cannot start up.

---

The more I look at this the more I think it may just be best to persist the whole db directory, because only persisting the client and server files might cause the tag mappings (also stored in the db directory) to be lost.
Since the sqlite3.exe is just a vanilla utility packed in with hydrus, but not actually used by hydrus, it should be fine to leave it in the persist.
```suggestion
    "bin": [
        [
            "client.exe",
            "hydrusclient"
        ],
        [
            "server.exe",
            "hydrusserver"
        ]
    ],
    "shortcuts": [
        [
            "client.exe",
            "hydrus client"
        ],
        [
            "server.exe",
            "hydrus server"
        ]
    ],
    "persist": "db",
```